src/test/java/lingogo/logic/commands/TestCommandTest.java

```diff
@@ -97,6 +110,8 @@ public void execute_validIndexFilteredListWrongGivenPhrase_success() {
         Model expectedModel = new ModelManager(model.getFlashcardApp(), new UserPrefs());
         showFlashcardAtIndex(expectedModel, INDEX_SECOND_FLASHCARD);
 
+        System.out.println(model.getFlashcardApp().getFlashcardList());
+
```

Comment:
If the print statement is necessary, use the `Logger` so that the code is consistent